tools/ARIAtools/extractProduct.py

```diff
@@ -1677,23 +1791,28 @@ def gacos_correction(full_product_dict, gacos_products, bbox_file,
                 # applicable)
                 elif '-' in tropo_rsc_dict['TIME_OF_DAY'][0]:
                     tropo_rsc_dict['TIME_OF_DAY'] = [
-                        datetime.datetime.strptime(m[:10] + '-' +
-                        m[11:].split('.')[0] + '-'
-                        + str(round(float('0.' + m[11:].split('.')[1]) * 60)),
-                        "%Y-%m-%d-%H-%M")
-                        for m in tropo_rsc_dict['TIME_OF_DAY']
-                    ]
+                        datetime.datetime.strptime(
```

Comment:
These expressions for `tropo_rsc_dict['TIME_OF_DAY']` are all too complicated and should be refactored so less expressions are being done inline

Reply:
This whole `gacos_correction` function will go away completely in a future PR. I want to remove that, plus other vestigial functions/scripts then.